tests/chainer_tests/distributions_tests/test_bernoulli.py

```diff
@@ -0,0 +1,39 @@
+from chainer import distributions
+from chainer import testing
+import numpy
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(3, 2), (1,)],
+    'is_variable': [True, False],
+    'sample_shape': [(3, 2), ()],
+}))
```

Comment:
Add a boolean parameter to test with `p` that contains `0` and `1`.
(You don't have to add it to the KL divergence test.)

Reply:
For now, I don't expect `p` whose type is `bool`, and we get error when boolean `p` is used (Because `F.log` does not support bool.).
And scipy.stats also don't support boolean `p`.
I think we don't have to support boolen `p`. How about this?

Reply:
What I tried to say is
```
if self.p_has_extreme_values:
    some_indices = ...
    p[some_indices] = np.random.randint(2, size=...)
```